project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts

```diff
@@ -76,10 +76,9 @@ export class EventDetailComponent implements OnInit {
   }
 
   get isenrollFlow() {
-    if (this.eventData) {
+    if (this.eventData && this.enrollFlowItems.length) {
```

Comment:
please do check for this.enrollFlowItems first then length

Reply:
enrollFlowItems is defined as array